src/sentry/tasks/derive_code_mappings.py

```diff
@@ -26,41 +22,27 @@
 logger = logging.getLogger("sentry.tasks.derive_code_mappings")
 
 
-@instrumented_task(
```

Comment:
The changes in #40882 will make it so we only queue these tasks for a single event per project. We don't need to support multiple organizations or projects.